virtualenv_tools.py

```diff
@@ -91,9 +94,9 @@ def update_script(script_filename, old_path, new_path):
         return
 
     args[0] = new_bin
-    lines[0] = '#!%s\n' % ' '.join(args)
+    lines[0] = b'#!%s\n' % b' '.join(args)
```

Comment:
in python3 this uses a python3.5+ feature, 3.4 isn't end-of-lifed yet so maybe we should use more compatible syntax?  Though yelp doesn't have any py34 left does it?